src/lib/utils.ts

```diff
@@ -365,25 +365,33 @@ export const purgeDefaultKeyValueStore = async () => {
 	await Promise.all(deletePromises);
 };
 
-export const outputJobLog = async (job: ActorRun | Build, timeout?: number) => {
+export const outputJobLog = async ({
+	job,
+	timeout,
+	apifyClient,
+}: {
+	job: ActorRun | Build;
+	timeout?: number;
```

Comment:
nit: we could follow the naming convention with `timeoutMillis` 

(now that it's not a positional parameter and we use the name on the call sites, it just shows more)